packages/styles/src/mixins/_button.scss

```diff
@@ -20,10 +20,13 @@
 
 @mixin button-size($size: md) {
   min-height: map.get(button.$btn-height-map, $size);
-  padding: 0 map.get(button.$btn-padding-x-map, $size);
   font-size: map.get(button.$btn-font-size-map, $size);
   gap: map.get(button.$btn-padding-x-map, $size) * 0.5;
 
+  &:not(.btn-link, .btn-tertiary) {
```

Comment:
I'm not sure this extra selector is necessary because it increases CSS output and makes padding harder to override if ever needed. What do you think?

Reply:
So, I'm not sure what's the best here. I need to increase specificity at some point to override the specific padding added in the sizes. The specificity is the same for both classes, but the size classes arrived later in the cascade.

I could as well:
* Set the important flag to the padding on `.btn-link, .btn-tertiary`.
* Increase specificity of the padding by targeting `.btn.btn-link, .btn.btn-tertiary`
* …

Reply:
`:where()` [takes a list of selectors as an argument](https://developer.mozilla.org/en-US/docs/Web/CSS/:where). It is not the case here, right?

Tertiary buttons already have overridden padding in the [current Design System](https://design-system.post.ch/#/bootstrap-samples/buttons). Is there something that has changed that we need to increase the specificity or is this a preventative measure?

Reply:
> :where() [takes a list of selectors as an argument](https://developer.mozilla.org/en-US/docs/Web/CSS/:where). It is not the case here, right?

I'm not sure what do you mean. [Pseudo-classes are selectors](https://www.w3.org/TR/selectors-4/#pseudo-classes), no? 
It seems to work, at least with the right displayed specificity, on WebKit:
![zV0ZoWBzQ4](https://github.com/swisspost/design-system/assets/12294151/7c4e096e-f313-458b-91bf-20b5afe6e1f9)

> Tertiary buttons already have overridden padding in the [current Design System](https://design-system.post.ch/#/bootstrap-samples/buttons). Is there something that has changed that we need to increase the specificity or is this a preventative measure?

It is true, but only with the version without an additional class to specify the size. In this case, the padding comes from the additional class, and it's the same for primary, secondary and tertiary, which is not the case for the latest variant according to Figma.